.github/workflows/npm-publish.yml

```diff
@@ -19,3 +19,4 @@ jobs:
           token: ${{ secrets.NPM_TOKEN }}
           check-version: true
           dry-run: false
+          tag: "beta"
```

Comment:
Temporarily changing the tag to "beta" - by default it is "latest". File storage and CSV features have not been completely merged into the `release` branch yet so the CLI options for those features are not completely functional, releasing as beta to indicate this.